eng/common/cross/build-rootfs.sh

```diff
@@ -256,15 +266,19 @@ if [[ "$__CodeName" == "alpine" ]]; then
       -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
       add $__AlpinePackages
 
-    $__ApkToolsDir/apk-tools-$__ApkToolsVersion/apk \
-      -X http://dl-cdn.alpinelinux.org/alpine/edge/main \
-      -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
-      add $__AlpinePackagesEdgeMain
+    if [[ -n "__AlpinePackagesEdgeMain" ]]; then
```

Comment:
@janvorli, the `$` is missing.
```suggestion
    if [[ -n "$__AlpinePackagesEdgeMain" ]]; then
```

Reply:
Oops, thank you for spotting it. Fixed in the last commit.